libs/api/domains/cms/src/lib/search/contentful.service.ts

```diff
@@ -170,6 +170,41 @@ export class ContentfulService {
     return _.flatten(chunkedChanges)
   }
 
+  /**
+   * Search for entries which have a field linking to a specific entry.
+   * Useful to finding root/parent of an entry
+   *
+   * @param linkId
+   * @param locale
+   * @private
+   */
+  private async linksToEntry(
+    linkId: string,
+    locale: string,
+  ): Promise<Entry<any>[]> {
+    const data = await this.contentfulClient.getEntries({
+      include: this.defaultIncludeDepth,
+      // eslint-disable-next-line @typescript-eslint/camelcase
+      links_to_entry: linkId,
+      locale: this.contentfulLocaleMap[locale],
+    })
+    return data.items
+  }
+
+  /**
+   * Alias of linksToEntry
+   *
+   * @param linkId
+   * @param locale
+   * @private
+   */
+  private async getParentEntries(
+    linkId: string,
+    locale: string,
+  ): Promise<Entry<any>[]> {
+    return this.linksToEntry(linkId, locale)
+  }
```

Comment:
Maybe we should rather just use the original function?
To me this increases the complexity of this file with little benefits